backend/vault/vault.go

```diff
@@ -0,0 +1,45 @@
+package vault
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/hashicorp/vault/api"
+	"github.com/heetch/confita/backend"
+)
+
+type vaultAPI interface {
```

Comment:
This is no longer necessary now since you have an integration test

Reply:
Nice catch ! Fixed.